_includes/configuration/formats.md

```diff
@@ -1,20 +1,32 @@
 ## formats
 
-The format option enables you to override and add custom **formats** to the editor.
+The format options are used to override and add custom **formats** to the editor.
 
-A format is a style that gets applied to text when you press, for example, the bold button inside the editor. TinyMCE is equipped with a text formatting engine that enables you to precisely specify what it should produce when the user clicks (in this example) the bold button.
+For example, a format is a style that gets applied to text when the bold button is enabled inside the editor. TinyMCE is equipped with a text formatting engine that allows specifying what it should produce when the user clicks (in this example) the bold button.
 
 Check out the [custom formats example]({{ site.baseurl }}/demo/format-custom/) for a demonstration of this option.
 
-Formats is an object where the values is either an object with format options or an array of format variants. When applying a format the first item in this formats array will be applied but when matching the format when removing or updating the ui the other formats in the array is considered as well. So the first format is a kind of primary format and the rest is secondary formats.
+Formats are objects where the value is either an object with format options or an array of format variants. 
+
+When a format is applied the first item, the formats array will be applied. However, when trying to match the format to remove or update the UI, the other formats in the array is considered as well. So the first format is a kind of primary format, and the rest is secondary formats.
```

Comment:
- the rest is secondary formats

+ the rest are secondary formats

Reply:
Fixed